.github/workflows/performance.yml

```
@@ -30,6 +30,15 @@ concurrency:
 permissions: {}
 
 env:
+  # Environment variables for Docker and PHP-FPM configuration
+  PHP_FPM_UID: ${{ steps.get-php-fpm-uid.outputs.php-fpm-uid }}
+  PHP_FPM_GID: ${{ steps.get-php-fpm-gid.outputs.php-fpm-gid }}
+
```

These can be removed if we're keeping the "Configure environment variables" step the same.

```suggestion
  # Environment variables for Docker and PHP-FPM configuration
  PHP_FPM_UID: ${{ steps.get-php-fpm-uid.outputs.php-fpm-uid }}
  PHP_FPM_GID: ${{ steps.get-php-fpm-gid.outputs.php-fpm-gid }}

```